### DISCUSSION TITLE
Magnetized transfer - retrieving metadata

### DISCUSSION THREAD

**Discussion by casaoui86** [haugene/docker-transmission-openvpn#2263] [Q&A]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [ ] I have tried using dev branch

### Docker run config used

sudo docker run --name="transmission-openvpn" --privileged -d \
	-v /volume1/docker/transmission-openvpn_2/custom:/data \
	-v /etc/localtime:/etc/localtime:ro \
	-v /volume1/docker/transmission-openvpn_2/custom/default.ovpn:/etc/openvpn/custom/default.ovpn:ro \
	-v /volume1/docker/transmission-openvpn_2/custom/auth.txt:/etc/openvpn/custom/auth.txt:ro \
	-v /volume1/docker/transmission-openvpn_2/custom/key.pem:/etc/openvpn/custom/key.pem:ro \
	-v /volume1/docker/transmission-openvpn_2/custom/client.crt:/etc/openvpn/custom/client.crt:ro \
	-v /volume1/docker/transmission-openvpn_2/custom/ca.crt:/etc/openvpn/custom/ca.crt:ro \
	-v /volume1/docker/transmission-openvpn_2/resolv.conf:/etc/resolv.conf \
	-e OPENVPN_PROVIDER=custom \
	-e CREATE_TUN_DEVICE=true \
	-e OPENVPN_CONFIG=default \
	-e OPENVPN_USERNAME=********** \
	-e OPENVPN_PASSWORD=********* \
	-e WEBPROXY_ENABLED=false \
	-e LOCAL_NETWORK=192.168.1.0/24 \
	-e OPENVPN_OPTS="--inactive 3600 --ping 10 --ping-exit 60" \
	-e TRANSMISSION_WEB_UI=transmission-web-control \
	-e TZ=Canada/Eastern \
	-e PUID=1037 \
	-e PGID=100 \
	-p 9091:9091 \
	-p 8888:8888 \
	--log-driver json-file \
	--log-opt max-size=10m \
	--sysctl net.ipv6.conf.all.disable_ipv6=0 \
	--dns 8.8.8.8 --dns 8.8.4.4 \
	haugene/transmission-openvpn

### Current Behavior

[2022-05-19 15:42:46.864] Mayans+M+C+S04E05+720p+WEB+H264+PLZPROPER Scrape error: **Could not connect to tracker (announcer.c:1279)**
[2022-05-19 15:42:46.864] Mayans+M+C+S04E05+720p+WEB+H264+PLZPROPER **Retrying scrape in 3640 seconds. (announcer.c:1288)**
[2022-05-19 15:42:46.864] Mayans+M+C+S04E03+720p+WEB+H264+PECULATE **Scrape error: Could not connect to tracker (announcer.c:1279)**

### Expected Behavior

Torrent start downloading

### How have you tried to solve the problem?

- allready checked /etc/resolv.conf : 
# Generated by openvpn for interface tun0
nameserver 8.8.8.8
nameserver 8.8.4.4
- 

### Log output

Starting container with revision: 20877f1b168b6ff27fc58aeef40756572c562d47
WARNING: initial DNS resolution test failed
Creating TUN device /dev/net/tun
mknod: /dev/net/tun: File exists
Using OpenVPN provider: CUSTOM
Running with VPN_CONFIG_SOURCE auto
No bundled config script found for CUSTOM. Defaulting to external config
Downloading configs from https://github.com/haugene/vpn-configs-contrib/archive/main.zip into /tmp/tmp.5GCIab9WyB
Extracting configs to /tmp/tmp.HO4KMguxyw
ERROR: Could not find any configs for provider CUSTOM in downloaded configs
Cleanup: deleting /tmp/tmp.5GCIab9WyB and /tmp/tmp.HO4KMguxyw
No VPN configuration provided. Using default.
Modifying /etc/openvpn/custom/default.ovpn for best behaviour in this container
Modification: Point auth-user-pass option to the username/password file
Modification: Update/set resolv-retry to 15 seconds
sed: cannot rename /etc/openvpn/custom/sedb9LDTO: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedkbiB7W: Device or resource busy
Modification: Change tls-crypt keyfile path
/etc/openvpn/modify-openvpn-config.sh: line 66: /etc/openvpn/custom/default.ovpn: Read-only file system
sed: cannot rename /etc/openvpn/custom/sedFhIvk2: Device or resource busy
Modification: Set output verbosity to 3
sed: cannot rename /etc/openvpn/custom/sedEsekEa: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedJwfp2f: Device or resource busy
/etc/openvpn/modify-openvpn-config.sh: line 88: /etc/openvpn/custom/default.ovpn: Read-only file system
Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop
sed: cannot rename /etc/openvpn/custom/sedyD3hZi: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedPrqsVo: Device or resource busy
/etc/openvpn/modify-openvpn-config.sh: line 100: /etc/openvpn/custom/default.ovpn: Read-only file system
Setting OpenVPN credentials...
adding route to local network 192.168.1.0/24 via 172.17.0.1 dev eth0
Thu May 19 14:46:48 2022 WARNING: file '/etc/openvpn/custom/auth.txt' is group or others accessible
Thu May 19 14:46:48 2022 OpenVPN 2.4.7 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Jul 19 2021
Thu May 19 14:46:48 2022 library versions: OpenSSL 1.1.1f  31 Mar 2020, LZO 2.10
Thu May 19 14:46:48 2022 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Thu May 19 14:46:48 2022 Outgoing Control Channel Encryption: Cipher 'AES-256-CTR' initialized with 256 bit key
Thu May 19 14:46:48 2022 Outgoing Control Channel Encryption: Using 256 bit message hash 'SHA256' for HMAC authentication
Thu May 19 14:46:48 2022 Incoming Control Channel Encryption: Cipher 'AES-256-CTR' initialized with 256 bit key
Thu May 19 14:46:48 2022 Incoming Control Channel Encryption: Using 256 bit message hash 'SHA256' for HMAC authentication
Thu May 19 14:46:48 2022 TCP/UDP: Preserving recently used remote address: [AF_INET]xxxxxxxxxx:1194
Thu May 19 14:46:48 2022 Socket Buffers: R=[212992->212992] S=[212992->212992]
Thu May 19 14:46:48 2022 UDP link local: (not bound)
Thu May 19 14:46:48 2022 UDP link remote: [AF_INET]xxxxxxxx:1194
Thu May 19 14:46:49 2022 TLS: Initial packet from [AF_INET]xxxxxxxxxx:1194, sid=167a215d 2b1621cb
Thu May 19 14:46:49 2022 VERIFY OK: depth=1, CN=ChangeMe
Thu May 19 14:46:49 2022 VERIFY KU OK
Thu May 19 14:46:49 2022 Validating certificate extended key usage
Thu May 19 14:46:49 2022 ++ Certificate has EKU (str) TLS Web Server Authentication, expects TLS Web Server Authentication
Thu May 19 14:46:49 2022 VERIFY EKU OK
Thu May 19 14:46:49 2022 VERIFY X509NAME OK: CN=-4d33-4c21-9b00-f954a304da04
Thu May 19 14:46:49 2022 VERIFY OK: depth=0, CN=-4d33-4c21-9b00-f954a304da04
Thu May 19 14:46:50 2022 Control Channel: TLSv1.3, cipher TLSv1.3 TLS_AES_256_GCM_SHA384, 256 bit EC, curve: prime256v1
Thu May 19 14:46:50 2022 [-4d33-4c21-9b00-f954a304da04] Peer Connection Initiated with [AF_INET]xxxxxxxxxx:1194
Thu May 19 14:46:51 2022 SENT CONTROL [-4d33-4c21-9b00-f954a304da04]: 'PUSH_REQUEST' (status=1)
Thu May 19 14:46:51 2022 PUSH: Received control message: 'PUSH_REPLY,dhcp-option DNS 8.8.8.8,dhcp-option DNS 8.8.4.4,block-outside-dns,redirect-gateway def1,route-gateway 10.8.0.1,topology subnet,ping 15,ping-restart 120,ifconfig 10.8.0.7 255.255.255.0,peer-id 0,cipher AES-256-GCM'
Thu May 19 14:46:51 2022 Options error: Unrecognized option or missing or extra parameter(s) in [PUSH-OPTIONS]:3: block-outside-dns (2.4.7)
Thu May 19 14:46:51 2022 OPTIONS IMPORT: timers and/or timeouts modified
Thu May 19 14:46:51 2022 OPTIONS IMPORT: --ifconfig/up options modified
Thu May 19 14:46:51 2022 OPTIONS IMPORT: route options modified
Thu May 19 14:46:51 2022 OPTIONS IMPORT: route-related options modified
Thu May 19 14:46:51 2022 OPTIONS IMPORT: --ip-win32 and/or --dhcp-option options modified
Thu May 19 14:46:51 2022 OPTIONS IMPORT: peer-id set
Thu May 19 14:46:51 2022 OPTIONS IMPORT: adjusting link_mtu to 1624
Thu May 19 14:46:51 2022 OPTIONS IMPORT: data channel crypto options modified
Thu May 19 14:46:51 2022 Data Channel: using negotiated cipher 'AES-256-GCM'
Thu May 19 14:46:51 2022 Outgoing Data Channel: Cipher 'AES-256-GCM' initialized with 256 bit key
Thu May 19 14:46:51 2022 Incoming Data Channel: Cipher 'AES-256-GCM' initialized with 256 bit key
Thu May 19 14:46:51 2022 ROUTE_GATEWAY 172.17.0.1/255.255.0.0 IFACE=eth0 HWADDR=02:42:ac:11:00:03
Thu May 19 14:46:51 2022 TUN/TAP device tun0 opened
Thu May 19 14:46:51 2022 TUN/TAP TX queue length set to 100
Thu May 19 14:46:51 2022 /sbin/ip link set dev tun0 up mtu 1500
Thu May 19 14:46:51 2022 /sbin/ip addr add dev tun0 10.8.0.7/24 broadcast 10.8.0.255
Thu May 19 14:46:51 2022 /etc/openvpn/tunnelUp.sh tun0 1500 1552 10.8.0.7 255.255.255.0 init
Up script executed with tun0 1500 1552 10.8.0.7 255.255.255.0 init
Updating TRANSMISSION_BIND_ADDRESS_IPV4 to the ip of tun0 : 10.8.0.7
Using Transmission Web Control UI, overriding TRANSMISSION_WEB_HOME
Updating Transmission settings.json with values from env variables
Using existing settings.json for Transmission /data/transmission-home/settings.json
Overriding bind-address-ipv4 because TRANSMISSION_BIND_ADDRESS_IPV4 is set to 10.8.0.7
Overriding download-dir because TRANSMISSION_DOWNLOAD_DIR is set to /data/completed
Overriding incomplete-dir because TRANSMISSION_INCOMPLETE_DIR is set to /data/incomplete
Overriding rpc-port because TRANSMISSION_RPC_PORT is set to 9091
Overriding watch-dir because TRANSMISSION_WATCH_DIR is set to /data/watch
sed'ing True to true
Enforcing ownership on transmission config directories
Applying permissions to transmission config directories
Setting owner for transmission paths to 1037:100
Setting permissions for download and incomplete directories
Mask: 002
Directories: 775
Files: 664
Setting permission for watch directory (775) and its files (664)

-------------------------------------
Transmission will run as
-------------------------------------
User name:   abc
User uid:    1037
User gid:    100
-------------------------------------

STARTING TRANSMISSION
Transmission startup script complete.
Thu May 19 14:46:52 2022 /sbin/ip route add xxxxxxxxxxxxxx /32 via 172.17.0.1
Thu May 19 14:46:52 2022 /sbin/ip route add 0.0.0.0/1 via 10.8.0.1
Thu May 19 14:46:52 2022 /sbin/ip route add 128.0.0.0/1 via 10.8.0.1
Thu May 19 14:46:52 2022 Initialization Sequence Completed

### HW/SW Environment

```markdown
- OS:lastest
- Docker:20.10.3
```

### Anything else?
Also i tried with an old version 3.0, but didn't work either 

Here is my default.ovpn config : 

client
dev tun
proto udp
remote xxxxxxxxxxx 1194
resolv-retry infinite
nobind
remote-cert-tls server
tls-version-min 1.2
cipher AES-256-CBC
auth SHA256
auth-nocache
askpass /etc/openvpn/custom/auth.txt
verb 3

<ca>
-----BEGIN CERTIFICATE-----
xxxxxxxxxxxxxx
-----END CERTIFICATE-----
</ca>
<cert>
-----BEGIN CERTIFICATE-----
xxxxxxxxxxx
BastSM+7IjLdayK5h1wfhrhV8+Rin38OiNwt
-----END CERTIFICATE-----
</cert>
<key>
-----BEGIN ENCRYPTED PRIVATE KEY-----
xxxxxxxxxxxxxx
-----END ENCRYPTED PRIVATE KEY-----
</key>
<tls-crypt>
#
# 2048 bit OpenVPN static key
#
-----BEGIN OpenVPN Static key V1-----
xxxxxxxx
-----END OpenVPN Static key V1-----
</tls-crypt>

disable-occ

**Comment by pkishino** [comment]
Migrated to discussion as this is not a container bug, this is a problem related to your setup, specifically DNS, see the second line in log.
You could try a different server/provider.
In the logs it seems the provider responds with some dns block option expected etc, could be they block external dns or so , check with your provider

**Comment by casaoui86** [comment]
Thanks for your input 
FYI i use Synology for Transmission-openvpn and RPI for Openvpn server 
so i disabled the option bellow from OpenVPN Server :

`**push "block-outside-dns"**`

and I tried a different DNS config : 

8.8.8.8, 8.8.4.4, 1.1.1.1, my box dns 192.168.1.1...
and didn't work

Here is the logs, and as you can see no DNS warning

**`Starting container with revision: 20877f1b168b6ff27fc58aeef40756572c562d47
Creating TUN device /dev/net/tun
mknod: /dev/net/tun: File exists
Using OpenVPN provider: CUSTOM
Running with VPN_CONFIG_SOURCE auto
No bundled config script found for CUSTOM. Defaulting to external config
Downloading configs from https://github.com/haugene/vpn-configs-contrib/archive/main.zip into /tmp/tmp.Cxqj53HUEO
Extracting configs to /tmp/tmp.HEhl7Aigkc
ERROR: Could not find any configs for provider CUSTOM in downloaded configs
Cleanup: deleting /tmp/tmp.Cxqj53HUEO and /tmp/tmp.HEhl7Aigkc
No VPN configuration provided. Using default.
Modifying /etc/openvpn/custom/default.ovpn for best behaviour in this container
Modification: Point auth-user-pass option to the username/password file
sed: cannot rename /etc/openvpn/custom/sedwAuVol: Device or resource busy
Modification: Change ca certificate path
sed: cannot rename /etc/openvpn/custom/sedstdftr: Device or resource busy
Modification: Change ping options
sed: cannot rename /etc/openvpn/custom/sedJDw8Vx: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedOkgoKD: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedu7aF7E: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedHlUfMJ: Device or resource busy
/etc/openvpn/modify-openvpn-config.sh: line 52: /etc/openvpn/custom/default.ovpn: Read-only file system
/etc/openvpn/modify-openvpn-config.sh: line 53: /etc/openvpn/custom/default.ovpn: Read-only file system
/etc/openvpn/modify-openvpn-config.sh: line 54: /etc/openvpn/custom/default.ovpn: Read-only file system
Modification: Update/set resolv-retry to 15 seconds
sed: cannot rename /etc/openvpn/custom/sedDabMKP: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedkbsmpU: Device or resource busy
/etc/openvpn/modify-openvpn-config.sh: line 66: /etc/openvpn/custom/default.ovpn: Read-only file system
Modification: Change tls-crypt keyfile path
sed: cannot rename /etc/openvpn/custom/sedqKF6E6: Device or resource busy
Modification: Set output verbosity to 3
sed: cannot rename /etc/openvpn/custom/sedOFIgP8: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedoE8i4d: Device or resource busy
/etc/openvpn/modify-openvpn-config.sh: line 88: /etc/openvpn/custom/default.ovpn: Read-only file system
Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop
sed: cannot rename /etc/openvpn/custom/sedntEAUj: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedcR7php: Device or resource busy
/etc/openvpn/modify-openvpn-config.sh: line 100: /etc/openvpn/custom/default.ovpn: Read-only file system
Setting OpenVPN credentials...
adding route to local network 192.168.1.0/24 via 172.17.0.1 dev eth0
Options error: In [CMD-LINE]:1: Error opening configuration file: /etc/openvpn/custom/default.ovpn
Use --help for more information.
Starting container with revision: 20877f1b168b6ff27fc58aeef40756572c562d47
Creating TUN device /dev/net/tun
Using OpenVPN provider: CUSTOM
Running with VPN_CONFIG_SOURCE auto
No bundled config script found for CUSTOM. Defaulting to external config
Downloading configs from https://github.com/haugene/vpn-configs-contrib/archive/main.zip into /tmp/tmp.bdyVwq0msg
Extracting configs to /tmp/tmp.j7UzJRvLB5
ERROR: Could not find any configs for provider CUSTOM in downloaded configs
Cleanup: deleting /tmp/tmp.bdyVwq0msg and /tmp/tmp.j7UzJRvLB5
No VPN configuration provided. Using default.
Modifying /etc/openvpn/custom/default.ovpn for best behaviour in this container
Modification: Point auth-user-pass option to the username/password file
sed: cannot rename /etc/openvpn/custom/sedUbiREi: Device or resource busy
Modification: Change ca certificate path
sed: cannot rename /etc/openvpn/custom/sedTW1lUr: Device or resource busy
Modification: Change ping options
sed: cannot rename /etc/openvpn/custom/sedD052by: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedT6M6QD: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedGiml1H: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedyHF0MK: Device or resource busy
/etc/openvpn/modify-openvpn-config.sh: line 52: /etc/openvpn/custom/default.ovpn: Read-only file system
/etc/openvpn/modify-openvpn-config.sh: line 53: /etc/openvpn/custom/default.ovpn: Read-only file system
/etc/openvpn/modify-openvpn-config.sh: line 54: /etc/openvpn/custom/default.ovpn: Read-only file system
Modification: Update/set resolv-retry to 15 seconds
sed: cannot rename /etc/openvpn/custom/sedI8IymR: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedDQWdEW: Device or resource busy
/etc/openvpn/modify-openvpn-config.sh: line 66: /etc/openvpn/custom/default.ovpn: Read-only file system
Modification: Change tls-crypt keyfile path
sed: cannot rename /etc/openvpn/custom/sedcuFxF5: Device or resource busy
Modification: Set output verbosity to 3
sed: cannot rename /etc/openvpn/custom/sedSJcM69: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedk6nBuc: Device or resource busy
/etc/openvpn/modify-openvpn-config.sh: line 88: /etc/openvpn/custom/default.ovpn: Read-only file system
Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop
sed: cannot rename /etc/openvpn/custom/sedwjJgOi: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedXb4vLl: Device or resource busy
/etc/openvpn/modify-openvpn-config.sh: line 100: /etc/openvpn/custom/default.ovpn: Read-only file system
Setting OpenVPN credentials...
adding route to local network 192.168.1.0/24 via 172.17.0.1 dev eth0
RTNETLINK answers: Operation not permitted
Options error: In [CMD-LINE]:1: Error opening configuration file: /etc/openvpn/custom/default.ovpn
Use --help for more information.
Starting container with revision: 20877f1b168b6ff27fc58aeef40756572c562d47
Creating TUN device /dev/net/tun
mknod: /dev/net/tun: File exists
Using OpenVPN provider: CUSTOM
Running with VPN_CONFIG_SOURCE auto
No bundled config script found for CUSTOM. Defaulting to external config
Downloading configs from https://github.com/haugene/vpn-configs-contrib/archive/main.zip into /tmp/tmp.TL1bGR70Zr
Extracting configs to /tmp/tmp.EdADV4ukDj
ERROR: Could not find any configs for provider CUSTOM in downloaded configs
Cleanup: deleting /tmp/tmp.TL1bGR70Zr and /tmp/tmp.EdADV4ukDj
No VPN configuration provided. Using default.
Modifying /etc/openvpn/custom/default.ovpn for best behaviour in this container
Modification: Point auth-user-pass option to the username/password file
sed: cannot rename /etc/openvpn/custom/sedQ9WJCh: Device or resource busy
Modification: Change ca certificate path
sed: cannot rename /etc/openvpn/custom/sedvuxsls: Device or resource busy
Modification: Change ping options
sed: cannot rename /etc/openvpn/custom/sedroeMby: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedIvgk2B: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedOW6e6H: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedFsGcLK: Device or resource busy
/etc/openvpn/modify-openvpn-config.sh: line 52: /etc/openvpn/custom/default.ovpn: Read-only file system
/etc/openvpn/modify-openvpn-config.sh: line 53: /etc/openvpn/custom/default.ovpn: Read-only file system
/etc/openvpn/modify-openvpn-config.sh: line 54: /etc/openvpn/custom/default.ovpn: Read-only file system
Modification: Update/set resolv-retry to 15 seconds
sed: cannot rename /etc/openvpn/custom/sed7pNa6Q: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sed51hToU: Device or resource busy
/etc/openvpn/modify-openvpn-config.sh: line 66: /etc/openvpn/custom/default.ovpn: Read-only file system
Modification: Change tls-crypt keyfile path
sed: cannot rename /etc/openvpn/custom/sedQ3wqP2: Device or resource busy
Modification: Set output verbosity to 3
sed: cannot rename /etc/openvpn/custom/sed4ru5hb: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedbCwgdc: Device or resource busy
/etc/openvpn/modify-openvpn-config.sh: line 88: /etc/openvpn/custom/default.ovpn: Read-only file system
Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop
sed: cannot rename /etc/openvpn/custom/sedjiPrIi: Device or resource busy
sed: cannot rename /etc/openvpn/custom/sedGUlF3m: Device or resource busy
/etc/openvpn/modify-openvpn-config.sh: line 100: /etc/openvpn/custom/default.ovpn: Read-only file system
Setting OpenVPN credentials...
adding route to local network 192.168.1.0/24 via 172.17.0.1 dev eth0
Mon May 23 07:18:01 2022 WARNING: file '/etc/openvpn/custom/auth.txt' is group or others accessible
Mon May 23 07:18:01 2022 OpenVPN 2.4.7 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Jul 19 2021
Mon May 23 07:18:01 2022 library versions: OpenSSL 1.1.1f  31 Mar 2020, LZO 2.10
Mon May 23 07:18:01 2022 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Mon May 23 07:18:01 2022 Outgoing Control Channel Encryption: Cipher 'AES-256-CTR' initialized with 256 bit key
Mon May 23 07:18:01 2022 Outgoing Control Channel Encryption: Using 256 bit message hash 'SHA256' for HMAC authentication
Mon May 23 07:18:01 2022 Incoming Control Channel Encryption: Cipher 'AES-256-CTR' initialized with 256 bit key
Mon May 23 07:18:01 2022 Incoming Control Channel Encryption: Using 256 bit message hash 'SHA256' for HMAC authentication
Mon May 23 07:18:01 2022 TCP/UDP: Preserving recently used remote address: [AF_INET]xxxxxxxxxxxx:1194
Mon May 23 07:18:01 2022 Socket Buffers: R=[212992->212992] S=[212992->212992]
Mon May 23 07:18:01 2022 UDP link local: (not bound)
Mon May 23 07:18:01 2022 UDP link remote: [AF_INET]xxxxxxxxxxxxxxx:1194
Mon May 23 07:18:01 2022 TLS: Initial packet from [AF_INET]xxxxxxxxxxx:1194, sid=85bacf9c 3e953cbe
Mon May 23 07:18:02 2022 VERIFY OK: depth=1, CN=ChangeMe
Mon May 23 07:18:02 2022 VERIFY KU OK
Mon May 23 07:18:02 2022 Validating certificate extended key usage
Mon May 23 07:18:02 2022 ++ Certificate has EKU (str) TLS Web Server Authentication, expects TLS Web Server Authentication
Mon May 23 07:18:02 2022 VERIFY EKU OK
Mon May 23 07:18:02 2022 VERIFY X509NAME OK: CN=ysnpi_53acbf64-4d33-4c21-9b00-f954a304da04
Mon May 23 07:18:02 2022 VERIFY OK: depth=0, CN=ysnpi_53acbf64-4d33-4c21-9b00-f954a304da04
Mon May 23 07:18:02 2022 Control Channel: TLSv1.3, cipher TLSv1.3 TLS_AES_256_GCM_SHA384, 256 bit EC, curve: prime256v1
Mon May 23 07:18:02 2022 [ysnpi_53acbf64-4d33-4c21-9b00-f954a304da04] Peer Connection Initiated with [AF_INET]xxxxxxxxxxxxx:1194
Mon May 23 07:18:03 2022 SENT CONTROL [ysnpi_53acbf64-4d33-4c21-9b00-f954a304da04]: 'PUSH_REQUEST' (status=1)
Mon May 23 07:18:03 2022 PUSH: Received control message: 'PUSH_REPLY,dhcp-option DNS 192.168.1.1,redirect-gateway def1,route-gateway 10.8.0.1,topology subnet,ping 15,ping-restart 120,ifconfig 10.8.0.7 255.255.255.0,peer-id 0,cipher AES-256-GCM'
Mon May 23 07:18:03 2022 OPTIONS IMPORT: timers and/or timeouts modified
Mon May 23 07:18:03 2022 OPTIONS IMPORT: --ifconfig/up options modified
Mon May 23 07:18:03 2022 OPTIONS IMPORT: route options modified
Mon May 23 07:18:03 2022 OPTIONS IMPORT: route-related options modified
Mon May 23 07:18:03 2022 OPTIONS IMPORT: --ip-win32 and/or --dhcp-option options modified
Mon May 23 07:18:03 2022 OPTIONS IMPORT: peer-id set
Mon May 23 07:18:03 2022 OPTIONS IMPORT: adjusting link_mtu to 1624
Mon May 23 07:18:03 2022 OPTIONS IMPORT: data channel crypto options modified
Mon May 23 07:18:03 2022 Data Channel: using negotiated cipher 'AES-256-GCM'
Mon May 23 07:18:03 2022 Outgoing Data Channel: Cipher 'AES-256-GCM' initialized with 256 bit key
Mon May 23 07:18:03 2022 Incoming Data Channel: Cipher 'AES-256-GCM' initialized with 256 bit key
Mon May 23 07:18:03 2022 ROUTE_GATEWAY 172.17.0.1/255.255.0.0 IFACE=eth0 HWADDR=02:42:ac:11:00:03
Mon May 23 07:18:03 2022 TUN/TAP device tun0 opened
Mon May 23 07:18:03 2022 TUN/TAP TX queue length set to 100
Mon May 23 07:18:03 2022 /sbin/ip link set dev tun0 up mtu 1500
Mon May 23 07:18:03 2022 /sbin/ip addr add dev tun0 10.8.0.7/24 broadcast 10.8.0.255
Mon May 23 07:18:03 2022 /etc/openvpn/tunnelUp.sh tun0 1500 1552 10.8.0.7 255.255.255.0 init
/etc/openvpn/tunnelUp.sh: line 48: /etc/resolv.conf: Read-only file system
chmod: changing permissions of '/etc/resolv.conf': Read-only file system
Up script executed with tun0 1500 1552 10.8.0.7 255.255.255.0 init
Updating TRANSMISSION_BIND_ADDRESS_IPV4 to the ip of tun0 : 10.8.0.7
Using Transmission Web Control UI, overriding TRANSMISSION_WEB_HOME
Updating Transmission settings.json with values from env variables
Using existing settings.json for Transmission /data/transmission-home/settings.json
Overriding bind-address-ipv4 because TRANSMISSION_BIND_ADDRESS_IPV4 is set to 10.8.0.7
Overriding download-dir because TRANSMISSION_DOWNLOAD_DIR is set to /data/completed
Overriding incomplete-dir because TRANSMISSION_INCOMPLETE_DIR is set to /data/incomplete
Overriding rpc-port because TRANSMISSION_RPC_PORT is set to 9091
Overriding watch-dir because TRANSMISSION_WATCH_DIR is set to /data/watch
sed'ing True to true
Enforcing ownership on transmission config directories
Applying permissions to transmission config directories
Setting owner for transmission paths to 1037:100
Setting permissions for download and incomplete directories
Mask: 002
Directories: 775
Files: 664
Setting permission for watch directory (775) and its files (664)

-------------------------------------
Transmission will run as
-------------------------------------
User name:   abc
User uid:    1037
User gid:    100
-------------------------------------

STARTING TRANSMISSION
Transmission startup script complete.
Mon May 23 07:18:04 2022 /sbin/ip route add xxxxxxxxxxxxx/32 via 172.17.0.1
Mon May 23 07:18:04 2022 /sbin/ip route add 0.0.0.0/1 via 10.8.0.1
Mon May 23 07:18:04 2022 /sbin/ip route add 128.0.0.0/1 via 10.8.0.1
Mon May 23 07:18:04 2022 Initialization Sequence Completed
`**

Any idea ? 

thnaks
